tests/Database/DatabaseMySqlSchemaGrammarTest.php

```diff
@@ -9,6 +9,9 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Schema\Grammars\MySqlGrammar;
 
+/**
+ * @group setTest
```

Comment:
What is this for?

Reply:
It was a comment to allow phpunit filtering. Forgot to take it out, sorry.